README.md

```diff
@@ -1,3 +1,129 @@
-extension-tao-lti
-=================
-extension to manage LTI services for TAO
+# TAO _taoLti_ extension
+
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+![GitHub](https://img.shields.io/github/license/oat-sa/extension-tao-lti.svg)
+![GitHub release](https://img.shields.io/github/release/oat-sa/extension-tao-lti.svg)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/oat-sa/extension-tao-lti.svg)
+
+> Extension to manage LTI services for TAO
+
+## Installation instructions
+
+These instructions assume that you have already a TAO installation on your system. If you don't, go to
+[package/tao](https://github.com/oat-sa/package-tao) and follow the installation instructions.
+
+If you installed your TAO instance through [package/tao](https://github.com/oat-sa/package-tao),
+`oat-sa/extension-tao-lti` is very likely already installed. You can verify this under _Settings -> Extension
+manager_, where it would appear on the left hand side as `taoLti`. Alternatively you would find it in
+the code at `/config/generis/installation.conf.php`.
+
+_Note, that you have to be logged in as System Administrator to do this._
+
+Add the extension to your TAO composer and to the autoloader:
+```bash
+composer require oat-sa/extension-tao-lti
+```
+
+Install the extension on the CLI from the project root:
+
+**Linux:**
+```bash
+sudo php tao/scripts/installExtension oat-sa/extension-tao-lti
```

Comment:
is it required to use `sudo` here?

Reply:
I don't know, it's copied from here: https://github.com/oat-sa/readme-templates/tree/master/extension-core/extension-tao-lti